pkgs/applications/audio/easyaudiosync/default.nix

```
@@ -0,0 +1,98 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, qtbase
+, qttools
+, spdlog
+, ffmpeg
+, taglib
+, wrapQtAppsHook
+, makeDesktopItem
+, copyDesktopItems
+,
+}:
+let
+  isDarwin = stdenv.isDarwin;
+  isLinux = stdenv.isLinux;
+in
+stdenv.mkDerivation rec {
+  pname = "easyaudiosync";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "complexlogic";
+    repo = "EasyAudioSync";
+    rev = "v${version}";
+    hash = "sha256-w98tj9BuixPhuDgwn74EYY0gvKH6kbfQmtg030RWRU0=";
+  };
+
+  patches =
+    [
+      ./0001-fix-project-name.patch
+      ./0002-fix-qt67-deprecated-methods.patch
+      ./0004-force-qt6.patch
+    ]
+    ++ lib.optionals isDarwin [
+      ./0003-fix-darwin-app.patch
+    ];
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook copyDesktopItems ];
+
+  buildInputs = [ qtbase qttools ffmpeg spdlog taglib ];
+
+  installPhase =
+    ''
+      runHook preInstall
+    ''
+    + (lib.optionalString isDarwin ''
+      mkdir -p $out/Applications
+      mv "easyaudiosync.app" "Easy Audio Sync.app"
+      cp -r "Easy Audio Sync.app" $out/Applications
+    '')
+    + (
+      lib.optionalString isLinux ''
+        install -Dm755 easyaudiosync $out/bin/easyaudiosync
+
+        for RES in 48 64 128 256; do
+          install -Dm755 "$src/assets/icons/easyaudiosync''${RES}.png" "$out/share/icons/hicolor/''${RES}x''${RES}/apps/${pname}.png"
+        done
+
+        install -Dm755 "$src/assets/icons/easyaudiosync.svg" "$out/share/icons/hicolor/scalable/apps/${pname}.svg"
```

```suggestion
          install -Dm755 "$src/assets/icons/easyaudiosync''${RES}.png" "$out/share/icons/hicolor/''${RES}x''${RES}/apps/easyaudiosync.png"
        done

        install -Dm755 "$src/assets/icons/easyaudiosync.svg" "$out/share/icons/hicolor/scalable/apps/easyaudiosync.svg"
```
substituting pname like this is an anti pattern